WordPress/gutenberg
# Make fonts test files use Core approach

## What?


Makes the fonts tests ready for Core merge by eliminating the need to rename the test files.

## Why?


To make it easier to merge these tests into Core.

WordPress Core's PHPUnit tests:
* are contained within `tests/phpunit/tests/` directory.
* do not add `-test.php` to the test filename.
* it uses `<directory suffix=".php">` to identify where the tests are located.

By moving the files into `phpunit/tests`, Gutenberg's tests will no longer need to add the `-test.php` suffix to the test filenames. This means the test files will not have to be renamed when merging them into WordPress Core.

## How?


* Relocates the fonts tests into `phpunit/tests/`
* Removes `-test` from each test filename.
* Removes testcase filename to `base.php` to match Core.
* Adds the directive to xml files.
* Adds 2 constants to identify where the location of `data` and `fixtures` directories.

## Testing Instructions

There should be no change in the number of tests that run before and after this PR.

To run all of the tests, run the following:
```
npm run test:unit:php
```
should produce the same results before and after this PR. (Try it on `trunk` and this PR.)

For the fonts tests only, run the following:

```
npm run test:unit:php:base -- --group fonts
```

which should produce the exact same results before and after this PR. (Try it on `trunk` and this PR.)
